package.json

```diff
@@ -200,6 +239,98 @@
+          "when": "view == peripheral-inspector.svd && viewItem == fieldWO",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.updateNode",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRW",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.copyValue",
+          "when": "view == peripheral-inspector.svd && viewItem == field",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.copyValue",
+          "when": "view == peripheral-inspector.svd && viewItem == fieldRO",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.copyValue",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRW",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.copyValue",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRO",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRW",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRO",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.forceRefresh",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRW",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.forceRefresh",
+          "when": "view == peripheral-inspector.svd && viewItem == register",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.forceRefresh",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRO",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.forceRefresh",
+          "when": "view == peripheral-inspector.svd && viewItem =~ /peripheral.*/",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.pin",
+          "when": "view == peripheral-inspector.svd && viewItem == peripheral",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.unpin",
+          "when": "view == peripheral-inspector.svd && viewItem == peripheral.pinned",
+          "group": "inline"
+        },
+        {
+          "command": "peripheral-inspector.svd.setFormat",
+          "when": "view == peripheral-inspector.svd",
+          "group": "navigation"
         }
       ],
```

Comment:
```suggestion
```

`view/item/context` is no longer used since this view is now rendered as a webview not a treeview.

Reply:
I have deleted the view/item/context, thank you for reviewing.